hypervisor/qemu/qemu_ppc64le.go

```diff
@@ -0,0 +1,63 @@
+// +build linux,ppc64le
+
+package qemu
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+
+	"github.com/hyperhq/runv/hypervisor"
+)
+
+func InitDriver() *QemuDriver {
+	cmd, err := exec.LookPath("qemu-system-ppc64le")
+	if err != nil {
+		return nil
+	}
+
+	return &QemuDriver{
+		executable: cmd,
+	}
+}
+
+func (qc *QemuContext) arguments(ctx *hypervisor.VmContext) []string {
+	if ctx.Boot == nil {
+		ctx.Boot = &hypervisor.BootConfig{
+			CPU:    1,
+			Memory: 256, // The minimum requirement for running a VM on PowerPC LE arch is 256M
+			Kernel: hypervisor.DefaultKernel,
+			Initrd: hypervisor.DefaultInitrd,
+		}
+	}
+	boot := ctx.Boot
+	qc.cpus = boot.CPU
+
+	var memParams, cpuParams string
+	if ctx.Boot.HotAddCpuMem {
+		memParams = fmt.Sprintf("size=%d,slots=1,maxmem=%dM", ctx.Boot.Memory, hypervisor.DefaultMaxMem) // TODO set maxmem to the total memory of the system
+		cpuParams = fmt.Sprintf("cpus=%d,maxcpus=%d", ctx.Boot.CPU, hypervisor.DefaultMaxCpus)           // TODO set it to the cpus of the system
```

Comment:
you can use boot.Memory instead of ctx.Boot.Memory,
boot.CPU instead of ctx.Boot.CPU.